Remotion/ObjectBinding/Web/Services/BocAutoCompleteReferenceValueSearchResult.cs

```diff
@@ -0,0 +1,39 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+namespace Remotion.ObjectBinding.Web.Services
+{
+  /// <summary>
+  /// The base search result of a <see cref="IBocAutoCompleteReferenceValueWebService.Search"/> call.
```

Comment:
```suggestion
  /// The search result of a <see cref="IBocAutoCompleteReferenceValueWebService"/>.<see cref="IBocAutoCompleteReferenceValueWebService.Search"/> call.
```

XmlDoc is funny that way. Unless you are fine with just the method name in the docs, you need to go the extra mile. (usually needed for referencing members on other types to make things readable).

And since it gets touched anyway, the 'base' qualifier isn't needed here.